apps/gitness/src/pages-v2/project/stores/labels-store.tsx

```diff
@@ -1,12 +1,30 @@
 import { create } from 'zustand'
 
-import { ILabelsStore, ILabelType } from '@harnessio/ui/views'
+import { ILabelsStore, ILabelType, LabelValueType } from '@harnessio/ui/views'
 
 export const useLabelsStore = create<ILabelsStore>(set => ({
-  labels: [],
+  spaceLabels: [],
+  repoLabels: [],
```

Comment:
why do we still need to maintain two separate arrays?
if the schema of the repo labels and space labels is the same, we can simplify this significantly by just maintaining one list, and overwriting if anything changes.